README.md

```diff
@@ -32,6 +32,8 @@ By analysing a few projects, most of the logic was identical in every single one
 
 A concern that came from the proposed solution was that every single website has different designs for a navbar or a drawer. Although this is true, their behaviour is usually the same. So, the solution should only deal with logic and behaviours while giving total freedom of the content that is rendered.
 
+⚠️ **Note:** If you are using this package on a `Layout` component that doesn't unmount/remount on each page change, you will probably need to listen to router events so that you make sure you close the drawer even if the page change is triggered by the browser history buttons.
```

Comment:
Should we document how it can done at least with Next.js? 🤔

Reply:
I didn't do it because this is an open package and that would be too specific. Either way, the main purpose of this kind of packages is to help us out at Moxy. Thus, I think I can add an example right below this note on how to do it with Next.js.

Reply:
Yeah, I thought as much. This module is primarily intended for us, that's why I've made the suggestion. Furthermore, with instructions in place on how to integrate with Next.js we can save some developer's time.